yaml_edit/lib/src/editor.dart

```diff
@@ -121,7 +121,6 @@ class YamlEditor {
   factory YamlEditor(String yaml) => YamlEditor._(yaml);
 
   YamlEditor._(this._yaml) {
-    ArgumentError.checkNotNull(_yaml);
```

Comment:
I've removed all calls to `ArgumentError.checkNotNull()` since these should be handled by Dart itself.